### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by jellywiz**
- No linter error
- Add Media Query
- Support Mobile and Desktop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,13 +13,27 @@
     <header>
         <nav class="navbar">
             <a class='logo'>Muzaffer Dler</a>
-            <img class="ulnavbar" src="./img/Union.svg" alt="Menu button">
+
+            <div class="desktop-nav">
```
Comment (ghost) [inline index.html:17]: - [x] Kindly align your social icons to the center of the headline section to match the figma design

image

### Checkout Figma 

image

- [x] Kindly fix all your buttons border thickness to match the figma design. You can use `border:1px solid color`

image

image


### Checkout figma

image

- [x] On the Figma design, you will notice that only the first card has `see project` button but all your cards have it. Kindly correct that. Also use different background images like in the Figma design.

image

### Checkout  figma

image

- [x] Kindly try to align your language and skills section items to the center. They are currently aligned to the right.

image

### Checkout figma

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @jellywiz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jellywiz) [reply index.html:17]: hi thanks for all of that great comment @frank1738 
the see project was on the mobile phone that's why I implemented on the desktop version if you would please check the mobile again and you will see for each project its the sample picture  and for the language and skills section:
image
it looks fine on my end

File: css/styless.css [hunk 2]
```diff
@@ -316,6 +320,7 @@ li {
   color: #fff;
   transition: all 0.5s ease-in;
   margin: auto;
+  border: 1px;
 }
```
Comment (alexpuente0) [inline css/styless.css]: - [OPTIONAL] You can center your button text on the multi-post stories by using the "Justify-content: center;" property on the ".rbb-btn" class.

before the edit:
image

after the edit:
image

File: css/styless.css [hunk 3]
```diff
@@ -597,3 +606,258 @@ li {
...
+    justify-content: center;
+    width: auto;
+  }
+
+  .form-header {
+    width: 455px;
+    height: 256px;
+    color: #262626;
+    padding-left: 300px;
+  }
```
Comment (alexpuente0) [inline css/styless.css:805]: - [OPTIONAL] You can try and remove the padding-left from this element, and adding some margin and width to the form (.form class) to center the items in the contact section.

before the edit:
image

after the edit:
image

File: index.html [hunk 4]
```diff
@@ -215,11 +234,11 @@ <h3 class="form-header">I'm always interested in hearing about new projects, so
...
             <button type="submit" class="informationbutton">Get in touch</button>
+            <hr class="social-media-line">
         </form>
     </section>
 
     <footer class="social-media-container">
-        <!-- <hr class="social-media-line"> -->
             <ul class="social-media-item1">
                 <li><img class="social-media-element" src="./img/github.png" alt=""></li>
                 <li><img class="social-media-element" src="./img/linkedin.png" alt=""></li>
```
Comment (alexpuente0) [inline index.html:244]: - [OPTIONAL] You seem to be missing a logo on the footer of the page (you have only four). Please consider adding it to your project before the next review.

image

File: index.html [hunk 5]
```diff
@@ -174,11 +193,11 @@ <h2 class="about-me-header">About Me</h2>
             </div>
         </div>
 
-        <hr>
+        <hr class="hr-aboutme">
```
Comment (alexpuente0) [inline index.html:197]: - [OPTIONAL] You can remove the margin form this `hr` tag and add it on the top and bottom of the `div` with resoucesection class. You can center this section using Justify-content: space-around; and margin: 150px auto; 

before the edit:
image

after the edit:
image

**Review by alexpuente0 — APPROVED** [review]
Hi @jellywiz! 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great Readme
- code is well organized

## Optional suggestions

I strongly recommend you apply this optional changes before your next code review, as it helps your UI to look more professional.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**